contracts/src/v0.8/functions/tests/v1_X/FunctionsTermsOfServiceAllowList.t.sol

```diff
@@ -247,6 +247,30 @@ contract FunctionsTermsOfServiceAllowList_GetAllAllowedSenders is FunctionsOwner
   }
 }
 
+/// @notice #getAllowedSendersInRange
+contract FunctionsTermsOfServiceAllowList_GetAllowedSendersInRange is FunctionsOwnerAcceptTermsOfServiceSetup {
+  function test_GetAllowedSendersInRange_Success() public {
+    // Send as stranger
+    vm.stopPrank();
+    vm.startPrank(STRANGER_ADDRESS);
+
+    address[] memory expectedSenders = new address[](1);
+    expectedSenders[0] = OWNER_ADDRESS;
+
+    assertEq(s_termsOfServiceAllowList.getAllowedSendersInRange(1, 1), expectedSenders);
```

Comment:
Let's test some more cases here such as adding some more allowed senders and then getting a larger range

Reply:
My 2 cents is I prefer to use zero indexing.

Reply:
+1 to that

Reply:
totally agree, I think the whole problem was that I based the implementation on subscriptions that have IDs and start at 1 instead of Idx here. Good catch, thanks! 🙇🏼